### PR TITLE
Add Doodlebug

### PR DESCRIPTION
**Pull request by choas**
Doodlebug from page 212

### CODE REVIEW

File: data/doodlebug.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  "volume": {
+    "value": 20,
+    "unit": "liters"
+  },
+  "boil_volume": {
+    "value": 25,
+    "unit": "liters"
+  },
+  "method": {
+    "mash_temp": {
```
Comment (sammdec) [inline data/doodlebug.json]: This now needs to have an array wrapped around it, due to some later recipes having multiple mash steps check out the template.json

**Comment by sammdec** [comment]
Apart from that one issue it all looks good to go thanks so much for contributing :+1:

**Comment by choas** [comment]
The TEMPLATE.json changed between my adding of doodlebug and the pull request (I was too slow). Adjusted it to "mash_temp as array".

**Comment by sammdec** [comment]
Looks great! Merging now! :+1:

**Pull request merged**